### ISSUE TITLE
geocache mod

### ISSUE THREAD

**Issue by thomasrudin** [pandorabox-io/pandorabox.io#88]
https://github.com/joshuamars/minetest-mod-geocache

**Comment by thomasrudin** [comment]
does not work :(

**Comment by Parasitoid** [comment]
Hi Thomas! 

I could test this mod only in singleplayer 0.4.17.1. I found that there is an unnecessary concatenation operator at the end of mixed string on a line 15. After removing those two dots this mod was loaded without any problems and I could place geo block (and that also makes "singleplayer" was added as a first sign on a list). Also a chat message was displayed any time I try to manipulate that geo block ("You already signed this..."). And that was all what I could do in singleplayer (0.4.17.1). What was wrong in your test? You wrote that this does not work. Did you tested this on server and something was wrong with other players actions?
